pytest-selenium/pages/content.py

```diff
@@ -36,6 +51,31 @@ def sidebar(self):
     def attribution(self):
         return self.Attribution(self)
 
+    @property
+    def section_url_within_attribution(self):
+        return self.find_element(*self._section_url_locator)
+
+    def click_content_overlay(self):
+        """Click anywhere in the content overlay
+
+        Print button is located at coordinatates (439, 164).
+        TOC close button is located at (16, 164).
+        So choosing (439, 164) to be the location in the content overlay.
+        Using actionchains to click on this position.
+        """
+        actionChains = ActionChains(self.driver)
+        actionChains.move_to_element_with_offset(self.driver.find_element_by_tag_name("body"), 0, 0)
+        actionChains.move_by_offset(439, 164).click().perform()
```

Comment:
@m1yag1 is this the best way to test this? the hardcoded pixel values make me nervous

Reply:
@TomWoodward 
Mike suggested this as an alternative method instead of hardcoding the pixels:
1. Calculate the sidebar width/height & window width/height
2. Use random function to select a random (x, y) coordinate 
3. Use action chains to click on the (x,y) coordinate

Do you want me to update this test with this suggestion?

cc @m1yag1

Reply:
@TomWoodward updated the test to use the random click as mentioned above